pkg/bastion/shell.go

```diff
@@ -2132,7 +2132,16 @@ GLOBAL OPTIONS:
 						if err := myself.CheckRoles([]string{"admin"}); err != nil {
 							return err
 						}
-
+						if err := dbmodels.UserKeysByIdentifiers(db, c.Args()).Find(&dbmodels.UserKey{}).Error; err != nil {
```

Comment:
@jwessel This error can be some internal/server error, maybe we should check is the type of error no found with 
errros.Is(), this way we will be more foolproof, also we should wrap the errors with more context